x-pack/plugins/fleet/server/saved_objects/migrations/security_solution/to_v8_11_0.ts

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { SavedObjectUnsanitizedDoc } from '@kbn/core/server';
+
+import type { SavedObjectModelDataBackfillFn } from '@kbn/core-saved-objects-server';
+
+import { omit } from 'lodash';
+
+import type { SavedObjectModelVersionForwardCompatibilityFn } from '@kbn/core-saved-objects-server';
+
+import type { PackagePolicy } from '../../../../common';
+
+export const migratePackagePolicyToV8110: SavedObjectModelDataBackfillFn<
+  PackagePolicy,
+  PackagePolicy
+> = (packagePolicyDoc) => {
+  if (packagePolicyDoc.attributes.package?.name !== 'endpoint') {
+    return { attributes: packagePolicyDoc.attributes };
+  }
+
+  const updatedPackagePolicyDoc: SavedObjectUnsanitizedDoc<PackagePolicy> = packagePolicyDoc;
+
+  const input = updatedPackagePolicyDoc.attributes.inputs[0];
+
+  if (input && input.config) {
+    const policy = input.config.policy.value;
+
+    policy.global_manifest_version = 'latest';
+  }
+
+  return {
+    attributes: {
+      inputs: updatedPackagePolicyDoc.attributes.inputs,
+    },
+  };
+};
+
+export const migratePackagePolicyEvictionsFromV8110: SavedObjectModelVersionForwardCompatibilityFn =
```

Comment:
@kevinlog One question though, should there be migrations to v8_10 in the backport?

Reply:
The goal of this backport is to ensure that we have the same `modelVersions` as in the `main` branch.  I think the naming here can be confusing because in the past we've named all of our migration scripts according to the stack release.

That being said, these migrations for `v8_11_0` will result in some policy fields that won't be used until the `8.11` release, but are backported here to keep those consistent and reduce risk for any Serverless deployments that are already up to date in `main`.

We do already have migrations to `v8_10` in the `8.10` line, we are adding a couple more here to: 1) Fix the original bug that we want to ship in `8.10.3` and backporting a bit extra to keep the `modelVersions` consistent between `8.10` and `8.11`